common4j/src/main/com/microsoft/identity/common4j/external/logging/IDetailedLoggerCallback.java

```diff
@@ -0,0 +1,12 @@
+package com.microsoft.identity.common4j.external.logging;
+
+/**
+ * An extension of ILoggerCallback - so that it prints the discarded log.
+ * This is for testing only (to verify that logs are actually discarded).
+ */
+interface IDetailedLoggerCallback extends ILoggerCallback {
+    /**
+     * Messages that are discarded by the loggers.
```

Comment:
When would log messages be discarded and why?

Reply:
PII when allowPII == false, or when log level is below what was set.